### DISCUSSION TITLE
Execute Python code on the fly and display results in Superset visualizations

### DISCUSSION THREAD

**Discussion by charithmadhuranga** [apache/superset#30652] [Ideas]
Analytics Extension implementation that expands superset's capabilities by allowing users to execute Python scripts and saved functions via superset 

**usecase**

Use Python for data cleaning and predictive algorithms inside superset.

**benifit**

Allows authoring calculated fields in Python.

This idea was taken from tableau's Tabpy framework

https://github.com/tableau/TabPy

**Comment by crazyproger** [comment]
This is the thing why we can't move from Zeppelin to Superset - lack of process-data-in-the-middle feature in superset.

**Comment by dmurphy123** [comment]
+1
